Notifier/UI/Windows/NotifierTrayIcon.cs

```diff
@@ -123,6 +114,8 @@ public void Hide()
         public void ShowActivity()
         {
             string tooltipText = Messages.NotifierTrayIcon_ShowActivity_Notifier;
+
+            // TODO: @harrwiss, why using a local method here also?
             void ShowBalloonTip()
```

Comment:
Can't remember anymore, but probably I moved it from the class to this place because its only used in that method anymore.

Reply:
Makes sense, but I think local methods should only exist when you rely on locally scoped variable :)